src/pf-notification-drawer/pf-notification-drawer.spec.js

```diff
@@ -0,0 +1,141 @@
+describe ('PatternFly Notification Drawer Component Tests', function () {
+  let drawer, btn;
+
+  function render(html) {
+    let template = document.createElement('template');
+    template.innerHTML = html;
+    document.body.appendChild(document.importNode(template.content, true));
+  }
+
+  beforeEach(function (done) {
+    let mo = new MutationObserver(function () {
+      mo.disconnect();
+      done();
+    });
+    mo.observe(document.body, { childList: true });
+    render(
+      '<button id="btn">Toggle Notification-Drawer</button>' +
+      '<pf-notification-drawer id="drawer" targetSelector="#btn">' +
+      ' <pf-notification-drawer-header drawerTitle="Notes"></pf-notification-drawer-header>' +
+      ' <pf-notification-drawer-accordion>' +
+      '   <pf-notification-drawer-accordion-panel panelTitle="Tab 1">' +
+      '     <pf-notification-drawer-accordion-panel-collapse>' +
+      '       <pf-notification-drawer-accordion-panel-body>' +
+      '         <p class="drawer-pf-notification unread">' +
+      '           <span class="drawer-pf-notification-message">Notfication 1</span>' +
+      '         </p>' +
+      '         <p class="drawer-pf-notification">' +
+      '           <span class="drawer-pf-notification-message">Notfication 2</span>' +
+      '         </p>' +
+      '       </pf-notification-drawer-accordion-panel-body>' +
+      '     </pf-notification-drawer-accordion-panel-collapse>' +
+      '   </pf-notification-drawer-accordion-panel>' +
+      '  </pf-notification-drawer-accordion>' +
+      '</pf-notification-drawer>'
+    );
+    btn = document.querySelector('#btn');
```

Comment:
In fact, we have the following code snippets to avoid collisions.
```js
  afterEach(function () {
    document.body.removeChild(drawer);
    drawer = null;
    document.body.removeChild(btn);
    btn = null;
  });
```
While, appending specific id will becoming the belt and braces. Very nice :)